### PR TITLE
python-r1 suite EAPI 8 support

### PR DESCRIPTION
**Pull request by mgorny**
CC @gentoo/python

### CODE REVIEW

File: eclass/multibuild.eclass [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
 # multiple 'variants' of a package (e.g. multilib, Python
 # implementations).
 
 case "${EAPI:-0}" in
-	0|1|2|3)
+	[0-3])
 		die "Unsupported EAPI=${EAPI:-0} (too old) for ${ECLASS}"
 		;;
-	4|5|6|7)
+	[4-8])
```
Comment (SoapGentoo) [inline eclass/multibuild.eclass:20]: why not tighten this further?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: eclass/multibuild.eclass (hunk 1 above)
Comment (mgorny) [reply eclass/multibuild.eclass:20]: I'm waiting for the last EAPI=5 consumers to go, then I will be able to remove a lot of old code.

Comment (SoapGentoo) [reply eclass/multibuild.eclass:20]: but you're still declaring EAPI 4 valid?

Comment (mgorny) [reply eclass/multibuild.eclass:20]: Yes because this commit is 'add new EAPI', 'remove old EAPIs' will come later.

**Pull request closed without merging**